csb/src/main/scala/edu/msstate/dasi/csb/DataParser.scala

```diff
@@ -43,9 +50,9 @@ object DataParser {
         respPort = pieces(5).toInt,
         proto = pieces(6),
         /* service = pieces(7), */
-        duration = pieces(8).toDouble,
-        origBytes = pieces(9).toLong,
-        respBytes = pieces(10).toLong,
+        duration = if (pieces(8) != "-") pieces(8).toDouble else 0.0,
+        origBytes = if (pieces(9) != "-") pieces(9).toLong else 0L,
+        respBytes = if (pieces(10) != "-") pieces(10).toLong else 0L,
```

Comment:
I see that we totally forgot the error handling for all conversions. I think it would be better to handle any int/long/double conversion inside a separate try/catch block, e.g.:
````scala
duration = try { pieces(8).toDouble } catch { case NumberFormatException => 0.0 }
````
It is OK to use `0` as a fallback value for numeric fields.